code/modules/mob/living/simple_animal/abnormality/he/snow_queen.dm

```diff
@@ -236,6 +237,16 @@
 	QDEL_IN(src, 10 SECONDS)
 	..()
 
+//Prevents gibbing during the duel.
+/mob/living/simple_animal/hostile/abnormality/snow_queen/gib()
+	if(arena_attacks)
+		return FALSE
+	else
+		..()
+
+/mob/living/simple_animal/hostile/abnormality/snow_queen/spawn_gibs()
+	return
+
```

Comment:
```suggestion
```
Redundant change given the changes in `_abnormality.dm`

Reply:
Done!